src/OpenTelemetry.Api.ProviderBuilderExtensions/Logs/OpenTelemetryDependencyInjectionLoggerProviderBuilderExtensions.cs

```diff
@@ -27,11 +27,11 @@ namespace OpenTelemetry.Logs;
 /// Contains extension methods for the <see cref="LoggerProviderBuilder"/> class.
 /// </summary>
 #if EXPOSE_EXPERIMENTAL_FEATURES
-    public
+public
 #else
     internal
 #endif
```

Comment:
This looks weird (no indentation on `public`, with indentation on `internal`) - is this a bug in the tool?

Reply:
It looks like code hidden within preprocessor directives in not formatted by dotnet format see https://github.com/dotnet/format/issues/783, I'd guess that the code exposed by the preprocessor directives isn't formatted properly due to directives breaking up definitions although I haven't looked into this to confirm it.

Reply:
Stay consistent with the indentation. Either keep it for both `public` and `internal` or remove it for both.